engine/src/main/java/org/camunda/bpm/engine/impl/batch/AbstractBatchJobHandler.java

```diff
@@ -39,6 +39,8 @@
 import org.camunda.bpm.engine.impl.util.ClockUtil;
 import org.camunda.bpm.engine.impl.util.JsonUtil;
 
+import static org.camunda.bpm.engine.impl.RestartProcessInstancesBatchConfigurationJsonConverter.PROCESS_DEFINITION_ID;
```

Comment:
❌ Not needed anymore:
```suggestion
```

Reply:
Hi @yanavasileva .
I've made the changes. I did try adding the test to confirm the instance and definitionId gets set in the job table, but I can't get he hold of restart job entry in this table. When I try to fetch the rows from Job table using - `List<Job> job = managementService.createJobQuery().list();` , I only get the monitor job. I think the job gets deleted soon after execution not sure though. 
I tried this code in `RestartProcessInstanceAsyncTest#shouldSetProcessInstanceAndDefinitionIdInHistoryJobLog`

Reply:
I added a comment below to point out where you perform the check for the runtime jobs.